distributions/application/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>gson</artifactId>
             <version>${gson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.codice.test</groupId>
+            <artifactId>junit-extensions</artifactId>
+            <version>${junit-ext.version}</version>
```

Comment:
✏️ Can we renamed this to codice-test.version since it will apply to all artifacts that comes from codice-test?

Reply:
❗️ I would also suggest only defining this, with scope `test` in the top-parent pom as done in DDF. It simplifies the need to define it everywhere. I would also add the other test dependencies as defined in DDF to simplify testing and ensure proper inter-dependencies and order.